### PR TITLE
Update installation example - Installation helm page of unity XT with supported K8s 

### PR DESCRIPTION
**Pull request by karthikk92**
# Description
Update installation example - Installation helm page of unity XT with supported K8s 

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/128 |

# Checklist:

- [ ] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [ ] Have you tested whether the hyperlinks are working properly?
- [x] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

**Review by shanmydell — DISMISSED** [review]

File: content/docs/csidriver/installation/helm/unity.md [hunk 1]
```diff
@@ -233,60 +233,60 @@ Procedure
...
-    |- Verifying that required namespaces have been created             Success
+    |--> Verifying minimum Kubernetes version                         Success
     |
-    |- Verifying that required secrets have been created                Success
+    |--> Verifying maximum Kubernetes version                         Success
     |
-    |- Verifying that required secrets have been created                Success
+    |- Verifying that required namespaces have been created           Success
+    |
+    |- Verifying that required secrets have been created              Success
```
Comment (AronAtDell) [inline content/docs/csidriver/installation/helm/unity.md:256]: Same as line 258. Is this how it should be?

**Review by AronAtDell — DISMISSED** [review]
Apart from my comment everything looks good. Thanks!

File: content/docs/csidriver/installation/helm/unity.md (hunk 1 above)
Comment (shanmydell) [reply content/docs/csidriver/installation/helm/unity.md:256]: @karthikk92 : why 256 and 258 lines are repeated is it a bug?

**Review by shanmydell — CHANGES_REQUESTED** [review]
Pls address review comments from Aron

File: content/docs/csidriver/installation/helm/unity.md (hunk 1 above)
Comment (karthikk92) [reply content/docs/csidriver/installation/helm/unity.md:256]: @shanmydell  same function is getting called for both empty and unity-creds as well 
image

Comment (karthikk92) [reply content/docs/csidriver/installation/helm/unity.md:256]: image
is this change looks ok?

Comment (shanmydell) [reply content/docs/csidriver/installation/helm/unity.md:256]: @karthikk92 : These changes need to be ported to all the drivers, Please track via a CSM issue. Message can be in simple terms like below.
Verifying that required secrets for creds have been created 
Verifying that required secrets for certs have been created

Comment (karthikk92) [reply content/docs/csidriver/installation/helm/unity.md:256]: @shanmydell @AronAtDell   taken care now

**Comment by karthikk92** [comment]
> Pls address review comments from Aron

Taken care @shanmydell

**Review by shanmydell — DISMISSED** [review]
LGTM

**Review by rensyct — DISMISSED** [review]

**Comment by shanmydell** [comment]
@karthikk92 : pls rebase the code

**Comment by mjsdell** [comment]
@karthikk92 please rebase

**Review by shanmydell — APPROVED** [review]

**Review by rensyct — APPROVED** [review]

**Pull request merged**